### DISCUSSION TITLE
Get favicon not found  "GET /favicon.ico HTTP/1.1" 404 Not Found

### DISCUSSION THREAD

**Discussion by MuhammadNizamani** [fastapi/fastapi#11385] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from fastapi.middleware.cors import CORSMiddleware


app = FastAPI(title="Quiz AI App Backend", version="0.0.1",docs_url='/')


app.add_middleware(
    CORSMiddleware,
    allow_origins=["*"],
    allow_methods=["*"],
    allow_headers=["*"],
    allow_credentials=True,
)


@app.get("/ping")
def health_check():
    """Health check."""

    return {"message": "Hello I am working!"}
```

### Description

When I am using this simple endpoint I  am get this in logs `"GET /favicon.ico HTTP/1.1" 404 Not Found`
But I check my other back-ends they are also showing same error .
I know what does error is say but I don't know how can solve this. and if anyone facing same issue tell me how did you solved it. 
Thanks
![fastapi issue](https://github.com/tiangolo/fastapi/assets/49721249/81b88f32-71b1-4e04-ad96-8dfeaaf95d88)

### Operating System

Linux

### Operating System Details

Distributor ID:	Zorin
Description:	Zorin OS 16.3
Release:	16
Codename:	focal

### FastAPI Version

0.110.0

### Pydantic Version

2.6.4

### Python Version

Python 3.10.14

**Comment by jgould22** [comment, marked as answer]
This is your browser automatically making a request looking for a favicon to put on the tab (and other places)

You can just ignore them if you don't care about the favicon.

If you do care you host a file at that path with that name using https://fastapi.tiangolo.com/tutorial/static-files/ or set https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel#icon in your html to tell the browser to check somewhere else.

**Reply by YuriiMotov** [reply]
I don't think it's a good idea to host static files at a root path with static-files (Or is everything okay with it?). And since the content type of the endpoint is 'json', you can't specify the path with `<link rel="icon" href="favicon.ico" />`.
So, I think the solution here is to create the endpoint to return favicon:
```
favicon_path = 'favicon.ico'  # Adjust path to file

@app.get('/favicon.ico', include_in_schema=False)
async def favicon():
    return FileResponse(favicon_path)
```

**Reply by MuhammadNizamani** [reply]
@YuriiMotov  the solution you provided it got this error 
**FileNotFoundError: [Errno 2] No such file or directory: 'favicon.ico'
RuntimeError: File at path favicon.ico does not exist.**

**Reply by YuriiMotov** [reply]
You have to create this file, put it somethere on you server and specify correct path (`favicon_path =`)

**Reply by MuhammadNizamani** [reply]
@YuriiMotov  sir I don't want favicon.ico

**Comment by jeosol** [comment]
Thanks @YuriiMotov  for the solution. For anyone using this approach remember to import from FileResponse as shown below:
```python
from fastapi.responses import FileResponse
